### DISCUSSION TITLE
how to get the reverse relations of a reference?

### DISCUSSION THREAD

**Discussion by scil** [tinacms/tinacms#2895] [Help]
for example,    
`article A` has a `author` reference whose value is `Job`  
`article B` has a `author` reference whose value is `Job` ，also.

Then, is there any way to get articles A and B for `job` ?

**Comment by kldavis4** [comment]
yes this is currently supported. You'll need to specify a filter like:
```
{
    "filter": {
        "author": {
            "authors": {
                "name": {
                    "eq": "Job"
                }
            }
        }
    }
}
```

where `author` is the name of the field on your article, `authors` is the name of the collection which `author` field references, and Job is the value of the `name` field on the `authors` collection

**Reply by scil** [reply]
Thank you.  

Based on reference, when editing article A, there is a dropdown menu to select author.   
On the author's side, is there any available ui to show the articles of the author? even more, to add/remove artilces for this author?

    article A has a author reference whose value is Job
    article B has a author reference whose value is Job ，also.
